.travis.yml

```diff
@@ -111,7 +111,9 @@ before_install:
     else
       brew update
       brew tap homebrew/gui
-      brew install python libpng ffmpeg imagemagick mplayer ccache
+      brew install python3 libpng ffmpeg imagemagick mplayer ccache
+      # make 'python' mean 'pyhon3'
```

Comment:
python3

Reply:
It would not be a PR from me with out a misspelling!